src/components/scenes/AddTokeScene.js

```diff
@@ -71,52 +72,46 @@ export class AddToken extends Component<AddTokenProps, State> {
           <View style={styles.instructionalArea}>
             <Text style={styles.instructionalText}>{s.strings.addtoken_top_instructions}</Text>
           </View>
-          <View style={styles.formArea}>
```

Comment:
Where did these styles go? They never existed, which means the `View` doesn't need to be here either.

Reply:
Also, visually it looks the same.